enhancements/platform-validation.md

```diff
@@ -160,102 +160,81 @@ It supports Kepler-Prometheus association by binding Kepler's specific service m
 make build-manifest OPTS="PROMETHEUS_DEPLOY"
 ```
 
-(2) Change the Prometheus service as `NodePort` type and get the accessible port.
+(2) Prometheus community provides [REST APIs](https://prometheus.io/docs/prometheus/latest/querying/api/) and [client-go](https://github.com/prometheus/client_golang/blob/main/api/prometheus/v1/api.go) libraries for specific queries.
 
-```
-kubectl patch svc prometheus-k8s -n monitoring --type='json' -p '[{"op":"replace","path":"/spec/type","value":"NodePort"}]'
-```
-```
-$kubectl get svc prometheus-k8s -n monitoring
-NAME             TYPE       CLUSTER-IP    EXTERNAL-IP   PORT(S)                         AGE
-prometheus-k8s   NodePort   10.96.205.9   <none>        9090:32517/TCP,8080:30331/TCP   5d9h
-```
 
-Tools such as `awk` could retrieve the accessible port:
-```
-$kubectl get svc prometheus-k8s -n monitoring | grep 9090 | awk '{print $5}' | awk -F ',' '{print $1}' | awk -F '/' '{print substr($1,6)}'
-32517
-```
+* Deploy from kepler-helm-chart.
 
-(3) Get the test node's IP.
+It by default does not support deploy kepler service monitor yet.
 
-```
-$ kubectl get nodes -o wide
-NAME                 STATUS   ROLES           AGE     VERSION   INTERNAL-IP   EXTERNAL-IP   OS-IMAGE             KERNEL-VERSION      CONTAINER-RUNTIME
-kind-control-plane   Ready    control-plane   5d10h   v1.25.3   172.19.0.2    <none>        Ubuntu 22.04.1 LTS   5.19.0-41-generic   containerd://1.6.9
-```
+* Deploy from kepler-operator.
 
-```
-$ kubectl get nodes -o wide |grep control-plane | awk '{print $6}'
-172.19.0.2
-```
+Kepler operator feature has some refactoring in 0.6.0 release period, we may cover this scenario in platform validation framework in later releases.
 
-(4) Retrieve the metrics with `cpu_architecture` label in Prometheus.
+We could leverage the `port forwarding` mechanism to expose accessible ports of `kepler-exporter` and `prometheus-k8s` sevices to the automation codes.
 
 ```
-$ curl --noproxy "*" -s http://172.19.0.2:32517/api/v1/label/cpu_architecture/values | jq -r ".data[]" | sort
-Sapphire Rapids
+kubectl port-forward --address localhost -n kepler service/kepler-exporter 9102:9102 &
+
+kubectl port-forward --address localhost -n monitoring service/prometheus-k8s 9090:9090 &
 ```
 
-* Deploy from kepler-helm-chart.
+To support validation cases comparison base, we are introcuding an indepentent RAPL-based energy collection and power consumption calculation tool called `validator` on Intel X86 BareMetal platforms.
```

Comment:
What is the problem with the current RAPL collector code? Can't you reuse it so that we avoid duplication and make sure that there is no mismatch between codes?

Reply:
There is no problem on current Kepler's RAPL code, but this PR is for validation cases design, so need to use independent way to check related data, otherwise it will be code test itself:)